### PR TITLE
[AAE-10480] : implementation find Identity Users By Group Id

### PR DESCRIPTION
**Pull request by scaccia**
- create new interface `IdentityService` with method `findUsersByGroupName`
- map new Feign Keycloak API: `/groups/{groupId}/members`
- implement `findUsersByGroupName` in `KeycloakManagementService`

### CODE REVIEW

File: activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java [hunk 1]
```diff
@@ -313,4 +314,12 @@ public void should_getGroupsClientRoleMapping() {
         assertThat(groups).extracting("name").contains("salesgroup");
     }
 
+
+    @Test
+    public void should_getUsers_by_groupId() {
+        List<KeycloakUser> users = keycloakClient.getUsersByGroupId(HR_GROUP_ID);
+
+        assertThat(users).hasSizeGreaterThanOrEqualTo(1);
```
Comment (miguelruizdev) [inline activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java:323]: could we check at least the name of the user? (I assume it is `hruser` right?)

File: activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/IdentityRuntimeService.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.identity;
+
+import java.util.List;
+import org.activiti.cloud.identity.model.User;
+
+public interface IdentityRuntimeService {
```
Comment (jesty) [inline activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/IdentityRuntimeService.java]: Could you rename to IdentityService, please can leave a comment on IdentityManagerService saying that that interface should be used only in Identity Adapter Service and not for other purposes, while for genreral purpose we should use this class.

**Review by jesty — CHANGES_REQUESTED** [review]

File: activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java (hunk 1 above)
Comment (miguelruizdev) [reply activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java:323]: also, `assertThat(users).hasSize(1)` should be more accurate to what we want to check

Comment (scaccia) [reply activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java:323]: There's a couple of issues about that.

1. Based on configuration `activiti-realm.json` I know there are at least one user in HR_GROUP. In addition, on group HR_GROUP are performed some logic in other tests. So the only thing I can test is hasSizeGreaterThanOrEqualTo
image

2. Unfortunately, as showed in previous image, the other groups haven't got id

If necessary I can create a new group and new user only for testing this issue.

Comment (miguelruizdev) [reply activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/test/java/org/activiti/cloud/services/identity/keycloak/KeycloakClientIT.java:323]: I see, now that I remember hruser and hradmin are inside that group, and maybe more
okay, it is safe to check it in the way you did
thanks for the clarification

**Review by miguelruizdev — APPROVED** [review]

**Review by jesty — APPROVED** [review]
LGTM

**Pull request merged**